thorlcr/slave/slavmain.cpp

```diff
@@ -209,6 +211,17 @@ class CJobListener : public CSimpleInterface
                         PROGLOG("verified mp connection to rest of slaves");
                 }
             };
+            rank_t slaveProc = queryNodeGroup().rank()-1;
+            StringBuffer slaveStr;
+            for (unsigned c=0; c<channelsPerSlave; c++)
+            {
+                unsigned o = slaveProc + (c * slavesPerNode);
+                if (c)
+                    slaveStr.append(",");
+                slaveStr.append(o+1);
+            }
+            PROGLOG("Slave log %u contains virtual slaves: %s", slaveProc+1, slaveStr.str());
+
```

Comment:
minor: I think, as with the master, I'd always issue this logging, even if channesPerSlaves=1 for consistency.